opentelemetry-exporter-geneva/opentelemetry-exporter-geneva/src/logs/exporter.rs

```diff
@@ -1,35 +1,25 @@
 use core::fmt;
-use geneva_uploader::Uploader;
+use geneva_uploader::client::GenevaClient;
 use opentelemetry_proto::transform::common::tonic::ResourceAttributesWithSchema;
 use opentelemetry_proto::transform::logs::tonic::group_logs_by_resource_and_scope;
-use opentelemetry_sdk::error::OTelSdkResult;
+use opentelemetry_sdk::error::{OTelSdkError, OTelSdkResult};
 use opentelemetry_sdk::logs::LogBatch;
 use std::sync::{atomic, Arc};
 
 /// An OpenTelemetry exporter that writes logs to Geneva exporter
 pub struct GenevaExporter {
     resource: ResourceAttributesWithSchema,
     _is_shutdown: atomic::AtomicBool,
-    uploader: Arc<Uploader>,
+    geneva_client: Arc<GenevaClient>,
```

Comment:
Do we need an `Arc` here?

```suggestion
    geneva_client: GenevaClient,
```